### DISCUSSION TITLE
求助关于luci上的mosdns 防止dns泄露在paopaodns上是否同样实现

### DISCUSSION THREAD

**Discussion by p1ayer222** [kkkgo/PaoPaoDNS#107] [问答和帮助] [answered]
![image](https://github.com/kkkgo/PaoPaoDNS/assets/35835318/41f20d47-22f5-46f3-addd-d1e5706b5eba)
这个图是luci 中的mosdns 界面截图，有个防止dns泄露的选项，搭配openclash很好用。
自从用上paopaodns后觉得很好用，paopaodns中也有mosdns，于是就想能不能用paopaodns替代luci中的mosdns，使用paopaodns作为openclash的上游。
经过实验，似乎不能防止dns泄露，请大大帮看看

**Comment by kkkgo** [comment, marked as answer]
参考文档的`USE_MARK_DATA`选项。
